hz: 20         # ループ周期 [Hz]
map_size: 4.0  # マップの一辺の長さ [m]
map_reso: 0.1  # マップの解像度 [m/cell]
yaw_reso: 0.1  # 角度解像度 [rad]
laser_step: 20 # 何本ずつレーザを見るか
